### ISSUE TITLE
Outdated documentation

### ISSUE THREAD

**Issue by yurug** [mirage/decompress#65]
`~proof` should be replaced by `~witness` in the following comment, right?

https://github.com/mirage/decompress/blob/37786262dfdea87ddb00167ec36550a4812e308e/lib/decompress.mli#L186

**Comment by dinosaure** [comment]
Yes, I will trupdate documentation. About diff between `proof` and `witness`, it's mostly an aesthetic update when `witness` is probably more convenience and understandable than `proof`. An update to the last API should be easy and make a new state of `decompress` is:

```ocaml
let t = Decompress.Zlib_deflate.default ~witness:Decompress.B.bytes level in
```

**Comment by yurug** [comment]
Thanks @dinosaure!

**Comment by dinosaure** [comment]
Fixed by #66
